### PR TITLE
Styling edits

### PR DESCRIPTION
**Pull request by interdigitize**

### CODE REVIEW

File: client/src/components/Nav.jsx [hunk 1]
```diff
@@ -67,7 +68,7 @@ class Navbar extends React.Component {
     return (
       <div>
         <AppBar
-          title="TrekTracker"
+          title={<TrekTracker/>}
```
Comment (jhjaylim) [inline client/src/components/Nav.jsx:71]: Being curious. I have never seen passing down entire component variable as props.

Is this intended?

Comment (interdigitize) [reply client/src/components/Nav.jsx:71]: @jhjaylim @nickelow Yea, it is intended. The material-ui nav component doesn't currently allow adding a image logo next to the title. I couldn't find a fitting glyphicon and don't think it is important enough to take the time to try to add an SVG, so I implemented a relevant version of this solution, https://github.com/callemall/material-ui/issues/5478

Comment (interdigitize) [reply client/src/components/Nav.jsx:71]: [
screen shot 2017-08-27 at 9 16 46 pm
](url) this is what it looks like. Totally cosmetic.

Comment (jhjaylim) [reply client/src/components/Nav.jsx:71]: good! merging!

**Pull request merged**